app/javascript/admin/controllers/modal_controller.js

```diff
@@ -8,6 +8,13 @@ export default class ModalController extends Controller {
     size: String,
   };
 
+  async closeModal() {
```

Comment:
by extracting `closeModal()`, we can call it from `app/javascript/admin/controllers/article_pinned_modal_controller.js` which inherits from this class